GVFS/GVFS.Common/PlaceholderListDatabase.cs

```diff
@@ -13,6 +13,7 @@ public class PlaceholderListDatabase : FileBasedCollection
         // - Not be a valid SHA-1 value (to avoid collisions with files)
         public const string PartialFolderValue = "                          PARTIAL FOLDER";
         public const string ExpandedFolderValue = "                         EXPANDED FOLDER";
+        public const string TombstoneFolderValue = "                        TOMBSTONE FOLDER";

```

Comment:
Yeah I was also wondering if this needed a version bump and an upgrader to get the all the tombstones from disk and add them to the placeholder database but since this should only matter during a git command I didn't.  What will bumping the minor version do as opposed to the major version?

Reply:
Oh I am glad you asked, I was wrong and it looks like this requires a major version bump.

```
        // The major version should be bumped whenever there is an on-disk format change that requires a one-way upgrade.
        // Increasing this version will make older versions of GVFS unable to mount a repo that has been mounted by a newer
        // version of GVFS.
        public int CurrentMajorVersion { get; }

        // The minor version should be bumped whenever there is an upgrade that can be safely ignored by older versions of GVFS.
        // For example, this allows an upgrade step that sets a default value for some new config setting.
        public int CurrentMinorVersion { get; }
```

If we wanted to avoid the major version bump then I think we'd have to go with the approach of a new database (that old VFS4G versions don't care about).

Reply:
>and an upgrader to get the all the tombstones from disk and add them to the placeholder database but since this should only matter during a git command I didn't.

I think we would only need this if we decided to handle the scenario of an interrupted git command followed by immediate upgrade of VFS4G.

That seems a bit obscure, but we should get some other people to weigh in as well.

/cc @jrbriggs